templates/admin.html

```diff
@@ -18,5 +18,22 @@ <h2>Home page content</h2>
 		<br />
 	</form>
 </section>
+<section>
+	<h2>Print All ID Cards</h2>
+	<form action=/cards/all>
+		<h3>Select semester for which to generate ID cards:</h3>
+		<select name="semester" size="{{ semesters|length }}">
+			{% for semester in semesters %}
+			<option value={{ semester }}{% if semester == current_semester%} selected="selected"{%endif%}>{{ semester }}</option>
+			{% endfor %}
+		</select>
+		<br />
+		<br />
+		<button type="submit">Print</button>
```

Comment:
Can you add the printer emoji, as used on the member list page, for consistency?  (I think it is `&#x1f5b6;`.)